sktime/forecasting/base/tests/test_fh.py

```diff
@@ -152,7 +153,7 @@ def test_check_fh_values_duplicate_input_values(arg):
 def test_check_fh_absolute_values_input_conversion_to_pandas_index(arg):
     """Test conversion of absolute horizons to pandas index."""
     output = ForecastingHorizon(arg, is_relative=False).to_pandas()
-    assert type(output) in VALID_INDEX_TYPES
+    assert is_in_valid_index_types(output) or is_integer_index(output)
```

Comment:
`is_relative_index_type or is_absolute_index_type`

Reply:
better probably `is_in_valid_index_types` once it´s changed (see below)

Reply:
done